src/lib.rs

```diff
@@ -333,6 +333,21 @@ where
     runner.run(f);
 }
 
+/// Run the given function under a random seed for one iteration.
+/// This effectively makes generating the random seed for each execution independent from
+/// `RandomScheduler`. Therefore, this can be used with a library (like proptest) that takes care of
+/// generating the random seeds.
+pub fn check_random_with_seed<F>(f: F, seed: u64)
```

Comment:
This needs to take the number of iterations as a parameter, so it should be:

`pub fn check_random_with_seed<F>(f: F, seed: usize, iterations: usize)`

Reply:
updated.